format/applebookmark/testdata/apple_bookmark.fqtest

```diff
@@ -0,0 +1,960 @@
+$ fq dv sample1.book 
```

Comment:
I've started to have the test files as `<FILE>` and `<FILE>.fqtest` pairs, think it makes a bit cleaner. Maybe good here?